arangod/Agency/State.cpp

```diff
@@ -95,8 +95,16 @@ inline static std::string stringify(index_t index) {
 bool State::persist(index_t index, term_t term, uint64_t millis,
                     arangodb::velocypack::Slice const& entry,
                     std::string const& clientId) const {
-  LOG_TOPIC("b735e", TRACE, Logger::AGENCY) << "persist index=" << index << " term=" << term
-                                   << " entry: " << entry.toJson();
+  LOG_TOPIC("b735e", TRACE, Logger::AGENCY)
+    << "persist index=" << index << " term=" << term << " entry: " << entry.toJson();
+
+#ifdef MAINTAINER_MODE // we should never get here
```

Comment:
the ifdef name is `ARANGODB_ENABLE_MAINTAINER_MODE:
```suggestion
#ifdef ARANGODB_ENABLE_MAINTAINER_MODE
```